spec/controllers/link_check_reports_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "spec_helper"
+require "services"
+
+describe LinkCheckReportsController, type: :controller do
+  describe "#create" do
+    let(:user) { FactoryGirl.create(:user) }
```

Comment:
Actually you can just use `create` here, no need for the `FactoryGirl`. (Will make it easier to upgrade to factory bot)